src/engraving/libmscore/splitMeasure.cpp

```diff
@@ -93,7 +93,7 @@ void Score::splitMeasure(Segment* segment)
         }
     }
 
-    // Make sure ties are the beginning the split measure are restored.
+    // Make sure ties to the beginning the split measure are restored.
```

Comment:
Sorry, still one word missing here: 
...the beginning **_of_** the split measure...